### DISCUSSION TITLE
Slimtoolkit community call/meeting

### DISCUSSION THREAD

**Discussion by Omkar0114** [slimtoolkit/slim#538] [Q&A] [answered]
Hey @kcq @iximiuz 
do we have weekly community calls or meetings to discuss, and explore the progress, and features of the Slim project? I am a student who is learning and looking to contribute to the cloud native ecosystem and the Slimtoolkit project excites me to explore more about containers. 

If there is then is there any calender for it? or not can we do weekly or biweekly calls to explore more on the slim repository?

Looking forward to your views on this. Thanks!

Currently, I am exploring Slim and going through the codebase trying to understand, and looking to solve good first issues  :-)

**Comment by kcq** [comment, marked as answer]
There are no scheduled community calls at this point in time, but it is something that is likely to happen after the project is done with the CNCF onboarding. Will be good to have.

**Reply by Omkar0114** [reply]
Cool. But Slimtoolkit is accepted to the CNCF sandbox, right? correct me if I am wrong!
